src/main/groovy/jenkins/automation/utils/GhUtils.groovy

```diff
@@ -8,8 +8,8 @@ class GhUtils {
 
 /**
  *
- * Utility method to watch a GitHub Enterprise or GitHub.com project and trigger on PR creation
- * This is a wrapper for the <a href="https://wiki.jenkins.io/display/JENKINS/GitHub+pull+request+builder+pluginGitHub">Pull Request Builder Plugin</a>.
+ * Utility method to watch a GitHub Enterprise or GitHub.com project and trigger on PR creation.<br/>
+ * This is a wrapper for the <a href="https://wiki.jenkins.io/display/JENKINS/GitHub+pull+request+builder+plugin">Pull Request Builder Plugin</a>.
```

Comment:
Think the original intent was _supposed_ to be

```
href="https://wiki.jenkins.io/display/JENKINS/GitHub+pull+request+builder+plugin">GitHub Pull Request Builder Plugin</a>.
```

Reply:
@champain roger will fix